code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -188,13 +188,14 @@ components:
         - message
       properties:
         status:
-          type: string
-          pattern: '^[1-5][0-9][0-9]$'
+          type: integer
+          minimum: 400
+          maximum: 599
```

Comment:
Looks ok, but why it cannot be described inside the response schema?

Reply:
It's how they are defining it in the guidelines. We might have to change it again later though, because they're waiting for people to vote. My suggestion is going forward with this format as this PR includes more fixes and once comonalities issue get resolved, I'll adapt it again to the result if it's needed.

Reply:
ok